ibc-apps/ics20-transfer/types/src/denom.rs

```diff
@@ -219,7 +219,10 @@ impl FromStr for TracePath {
         remaining_parts
             .is_none()
             .then_some(trace_path)
-            .ok_or_else(|| TokenTransferError::InvalidTrace(s.to_string()))
+            .ok_or_else(|| TokenTransferError::FailedToParseType {
+                expected: "TracePath".to_string(),
```

Comment:
The same comment that I left for `expected: "Coin".to_string()` applies here too.

Reply:
[2952291](https://github.com/cosmos/ibc-rs/pull/1325/commits/29522914999b02c31f3e933d7cfc273f2c24a916)